hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpDeleteCommandProcessor.java

```diff
@@ -48,7 +50,7 @@ public void handle(HttpDeleteCommand command) {
         textCommandService.sendResponse(command);
     }
 
-    private void handleMap(HttpDeleteCommand command, String uri) {
+    private void handleMap(HttpDeleteCommand command, String uri) throws UnsupportedEncodingException {
```

Comment:
why do we need to add this exception to the method? same for others

Reply:
leftover, removed